fastlane/Fastfile

```diff
@@ -1,30 +1,35 @@
-# This file contains the fastlane.tools configuration
-# You can find the documentation at https://docs.fastlane.tools
-#
-# For a list of all available actions, check out
-#
-#     https://docs.fastlane.tools/actions
-#
-# For a list of all available plugins, check out
-#
-#     https://docs.fastlane.tools/plugins/available-plugins
-#
-
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
+require 'fileutils'
 
 default_platform(:ios)
 
-platform :ios do
-  desc "Description of what the lane does"
-  lane :custom_lane do
-    # add actions here: https://docs.fastlane.tools/actions
-  end
+$ignore_file_path = '.xcovignore'
+
+before_all do
+	FileUtils.rm_rf("./UnitTestsReport")
+	FileUtils.rm_rf("./CodeCoverageReport")
 end
 
-lane :unit_tests do
-  run_tests(build_for_testing: true,
-            scheme: "Basic-Car-Maintenance",
-            only_testing: "Basic-Car-Maintenance-Tests",
-            device: "iPhone 15 Pro")
+platform :ios do
+	desc "Run tests and create a unit-test report"
+
+	lane :unit_tests do
+  		run_tests(
+			scheme: "Basic-Car-Maintenance",
+			only_testing: "Basic-Car-Maintenance-Tests",
+			clean: true,
+			code_coverage: true,
+			device: "iPhone 15 Pro",
+			skip_build: true,
+			xcodebuild_formatter: 'xcbeautify -qq --is-ci --renderer github-actions',
+			output_types: 'junit',
+			output_directory: "./fastlane/UnitTestsReport"
+		)
+
+		xcov(
+			scheme: "Basic-Car-Maintenance",
+			only_project_targets: true,
+			output_directory: "./fastlane/CodeCoverageReport"
```

Comment:
Does this need to be ignored?

Reply:
Not sure what you mean? `xcov` is the fastlane action that runs coverage for the project.

Reply:
Updated gitignore file to exclude reports from accidentally being committed to the repo